### PR TITLE
refactor(api-gen): show @input/@output aliases

### PR DESCRIPTION
**Pull request by JeanMeche**
Fixes angular/angular#56359

### CODE REVIEW

File: bazel/api-gen/rendering/transforms/code-transforms.ts [hunk 1]
```diff
@@ -313,7 +307,20 @@ function isOptionalMember(member: PropertyEntry): boolean {
 
 function getTags(member: PropertyEntry): string[] {
   return member.memberTags
-    .map((tag) => ANGULAR_PROPERTY_DECORATORS[tag] ?? tag)
+    .map((tag) => {
+      if (tag === 'input') {
+        !member.inputAlias || member.name === member.inputAlias
```
Comment (josephperrott) [inline bazel/api-gen/rendering/transforms/code-transforms.ts:312]: Should this be 
```suggestion
        return !member.inputAlias || member.name === member.inputAlias
```


I think as written we don't return any string at all on most paths through this function, maybe I am confused at what you are trying to do here

Comment (JeanMeche) [reply bazel/api-gen/rendering/transforms/code-transforms.ts:312]: Good catch !

**Review by josephperrott — APPROVED** [review]
LGTM

File: bazel/api-gen/rendering/transforms/code-transforms.ts [hunk 2]
```diff
@@ -313,7 +307,20 @@ function isOptionalMember(member: PropertyEntry): boolean {
...
+          : `@Input('${member.inputAlias}')`;
+      } else if (tag === 'output') {
+        return !member.outputAlias || member.name === member.outputAlias
+          ? '@Output()'
+          : `@Output('${member.outputAlias}')`;
+      } else if (tag === 'optional') {
+        return '';
+      }
+      return tag;
+    })
```
[the remark this answers is not in the document]
Comment (JeanMeche) [reply bazel/api-gen/rendering/transforms/code-transforms.ts:323]: Nested ternaries are less readable imho.

**Comment by josephperrott** [comment]
This PR was merged into the repository by commit c62e8953263ded837d9ed1d4fb2d072768e4482c.

The changes were merged into the following branches: main

**Pull request closed without merging**